### PR TITLE
Admin UI: Upgrade flutter version

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

**Review by tnotheis — APPROVED** [review]
I'll approve so you're able to merge. But maybe you should wait for one of the flutter guys to approve, as I don't know if anything is missing.

**Review by Siolto — APPROVED** [review]

**Pull request merged**
